### PR TITLE
Update dependencies for many samples

### PR DESCRIPTION
**Pull request by ehoogerbeets**

### CODE REVIEW

File: android/package.json [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
+    "name": "sample",
+    "description": "Sample localization project",
+    "repository": "git@github.com:iLib-js/ilib-loctool-samples.git",
+    "license": "Apache-2.0",
+    "version": "1.0.0",
+    "scripts": {
+        "loc": "loctool",
+        "debug": "node --inspect-brk node_modules/loctool/loctool.js",
+        "clean": "rm -rf *.xliff res/values-* res/values/strings-auto.xml",
+        "test": "echo success"
```
Comment (gouniLee) [inline android/package.json:11]: It means nothing. is there a reason to add it?

Comment (gouniLee) [inline android/package.json:11]: It means nothing. is there a reason to add it?

Comment (ehoogerbeets) [reply android/package.json:11]: npm-check-updates runs the "test" script to see if it broke anything before and after it upgrades the version numbers. It needs to be there in order to have it update automatically.

Comment (ehoogerbeets) [reply android/package.json:11]: Check it out: https://www.npmjs.com/package/npm-check-updates

**Review by gouniLee — APPROVED** [review]

**Pull request merged**